arch/arm/boot/dts/overlays/w1-gpio-pullup-overlay.dts

```diff
@@ -7,7 +7,7 @@
 
 	fragment@0 {
 		target-path = "/";
-		__overlay__ {
+		frag0: __overlay__ {
```

Comment:
Not needed.

Reply:
Arg, missed this one - one moment please...